src/olympia/files/decorators.py

```diff
@@ -54,7 +54,7 @@ def last_modified(request, obj, key=None, **kw):
 
 
 def etag(request, obj, key=None, **kw):
-    return _get_value(obj, key, 'md5')
+    return _get_value(obj, key, 'sha256')
```

Comment:
Also, Django still uses MD5 for Etag calculation and given any potential performance implications I don't see why we shouldn't. Etag only requires a "function with rare enough conflicts". Let's at least keep a closer eye on wether or not this implicates performance.